pkg/webhook/cluster/mutating_webhook.go

```diff
@@ -133,7 +133,7 @@ func (a *ManagedClusterMutatingAdmissionHook) addDefaultClusterSetLabel(managedC
 		return nil, status
 	}
 
-	labelPatch := newLabelJsonPatch()
+	labelPatch := newLabelJsonPatch(cluster.Labels)
```

Comment:
if it is add operation, why it needs to know the original label?

Reply:
The `add`  operation for `labels` will override the existing `labels`.